README.md

```diff
@@ -1,143 +1,415 @@
 
-## Orbit versus Funnel
+### Measuring gravity
 
-| Funnel | Orbit |
-|-----------|-----------|
-| Conversion | Adoption |
-| Value capture | Value creation |
-| Push people through | Pull people in | 
-| Based on single purchase event | Inclusive of entire lifecycle | 
-| Binary (leads vs. everyone else) | Non-binary (Orbit Levels) | 
+Love and reach are measured at the individual level, but gravity is measured at the group or community level. Gravity can also be measured for each orbit level.
 
+**Gravity should reflect things that are happening in the community:**
 
-## Gravity = Love x Reach
+- Gravity goes up when new members join
+- Gravity goes up when members participate more (increased love)
+- Gravity goes up when members make the community visible (increased reach)
+- Gravity goes down when we lose members or participation declines
 
-This is the fundamental equation of the Orbit Model. Think of it this way: everyone in your community has some amount of gravity. Some ability to attract others. The gravity that each person has is a product of two things: **love and reach**.
+A basic equation that takes those factors into account is:
 
-| Love | Reach |
-|-----------|-----------|
-| Love is their love for what you do. That includes expert knowledge in your technology, a high degree of satisfaction, and feeling like part of the tribe. | Reach is a measure of how well they can help spread the love. Developers with a lot of reach tend to be well-connected, respected by their peers, and have a passion for teaching and sharing. |
+```
+Gravity = Sum(Love * Reach) for each member
+```
 
+**The simplest way to calculate gravity is just to sum up the product of love and reach for each member.**
 
-## Orbit Levels
+Using the examples from above, here's what the gravity would look like.
 
-Love, reach, and gravity are important because they help you segment your community into different levels. We call these orbit levels. Having orbit levels enables you to give the right opportunity to the right person at the right time.
+| Member Name       | Love | Reach | Love \* Reach |
+| ----------------- | ---- | ----- | ------------- |
+| Avery Hartford    | 2    | 5     | 10            |
+| Laura Blight      | 6    | 3     | 18            |
+| Jennifer Lewis    | 7    | 3     | 21            |
+| Terrence Fletcher | 4    | 5     | 20            |
+| Marie Monzo       | 2    | 1     | 2             |
+|                   |      |       | ----          |
+| **Gravity**       |      |       | 71            |
 
-You can name your orbit levels what you want and have as many as you like, but to have a standard we can all reference, we describe 4 levels with the following names that reflect the shape of many modern communities.
+**Reach is multiplied by (not added to) love because an increase in a member's reach has the potential to make their impact higher across each place they participate.**
```

Comment:
> across each place they participate
Can you clarify? e.g. "in each community they participate in," or, "with each activity the complete in your community."

Reply:
Good call, added something